### DISCUSSION TITLE
Brave & Gmail

### DISCUSSION THREAD

**Discussion by KurisuD55** [GorvGoyl/ChatGPT-Writer#85] [Report Bugs and Issues]
Hey,  
I've changed the settings in Brave as per the instructions but still getting this error.  

Anyone able to assist?  
![Screenshot 2023-04-13 114613](https://user-images.githubusercontent.com/130621880/231625072-ada09159-e164-4f5e-be26-6bda1b54b339.png)

**Comment by GorvGoyl** [comment]
did you visit chat.openai.com to make sure you are logged in?

**Reply by KurisuD55** [reply]
Yep, all logged in :)
